tests/test_metadata_tools.py

```diff
@@ -59,6 +57,20 @@ def test_exif_attach(self):
         self.assertEqual("15.2.3", exif_dict['Software'])
         self.assertEqual(3, exif_dict['Orientation'])
 
```

Comment:
new unittest to test if an analagous iptc field is not overwritten by writing to exif, and vice versa.